db/institutions.yml

```diff
@@ -262,4 +262,13 @@ nyu:
   name: New York University
   state: NY
   zip: 10003
+qmul:
+  aka: [QMUL, Queen Mary University of London, Queen Mary College, QMC]
+  city: London
+  country: UK
+  schools: {physastro: The School of Physics and Astronomy}
+  departments: {cmmaterials: Centre for Condensed Matter and Materials Physics}
+  name: Queen Mary University of London
+  state: ''
```

Comment:
please a street address if you can find one.  The zip will be something like SW3 4ED